test/tests/functional/pbs_node_rampdown.py

```diff
@@ -607,7 +602,7 @@ def tearDown(self):
         TestFunctional.tearDown(self)
         # Delete managers and operators if added
         attrib = ['operators', 'managers']
-        self.server.manager(MGR_CMD_UNSET, SERVER, attrib)
+        self.server.manager(MGR_CMD_UNSET, SERVER, attrib, sudo=True)
```

Comment:
I think PBSTestSuite's teardown() reverts managers and operators.  Why is the test unsetting them?